integration-tests/tests/create-application-from-public-git-source.spec.ts

```diff
@@ -54,9 +54,13 @@ describe('Create Component from Public Git Source', () => {
     });
 
     it('Check Changing Resources', () => {
-      componentPage.expandDetails(componentName);
-      componentPage.setCpuByButton(cpuCount + 1, cpuUnit);
-      componentPage.setRam(ramValue, ramUnit);
+      cy.get(ComponentsPagePO.extractComponentName).then((innerText) => {
+        componentPage.componentName = innerText.text().trim();
+
+        componentPage.expandDetails(componentPage.componentName);
+        componentPage.setCpuByButton(cpuCount + 1, cpuUnit);
+        componentPage.setRam(ramValue, ramUnit);
```

Comment:
As the component name is now known outside the callback, I would move those lines outside. But that's more like my own feeling. If you want to leave it as it is, feel free.

Reply:
I'd say we leave it here because I spent a good amount of time trying to move these lines outside and make it work but it fails. Since cypress commands are async the `componentName` variable would hold the `undefined` value outside of the `.then()` block.

Reply:
Ok, thx for the explanation.